VariantShredding.md

```diff
@@ -110,18 +119,22 @@ Shredded values must use the following Parquet types:
 
 Primitive values can be shredded using the equivalent Parquet primitive type from the table above for `typed_value`.
 
-Unless the value is shredded as an object (see [Objects](#objects)), `typed_value` or `value` (but not both) must be non-null.
+Unless the value is shredded as an object (see [Objects](#objects)), exactly one of `typed_value` or `value` must be present in each row.
+
+NULL values in an optional group are encoded in the usual way, with a definition level that excludes
```

Comment:
I think that this is confusing because it draws a distinction between SQL NULL and how Parquet represents null values.